### PR TITLE
Feature/update resource page

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Removes all old design pattern resources
- Adds links to new design pattern mono repos
- Add link to new react workflow package

#### Screenshots:
image

### CODE REVIEW

**Comment by emclaug2** [comment]
In the story it says that the new design pattern repos should be a single line instead of split into 4.  Since each item in the list links out to a demo, I wasn't sure how this should look visually.  I split the single item into 4.  

Is there a design for the single InfoListItem that links out to all 4 design pattern repos?

**Comment by joebochill** [comment]
> In the story it says that the new design pattern mono repos should be condensed into a single line. Since each item in the list links out to a demo, I wasn't sure how this should look visually. I split the single item into 4.
> 
> Is there a design for the single InfoListItem that links out to all 4 design pattern repos?

Your interpretation is correct. I was expecting a single line item for each monorepo...not one for all four.

**Review by joebochill — DISMISSED** [review]
- The live example buttons should link to CodeSandbox, StackBlitz, or Snack
- Should the github links point to the master branch instead of dev? The other packages do.

**Comment by emclaug2** [comment]
> * The live example buttons should link to CodeSandbox, StackBlitz, or Snack
> * Should the github links point to the master branch instead of dev? The other packages do.

I'll update the demo links to point directly to the demo and update the github link to point to master.

**Comment by emclaug2** [comment]
Since `ExampleRow` and `PackageRow`s were becoming very similar, I've merged these components into the `ResourceRow`.   This keeps the row generation logic a lot more streamlined imo.

File: src/app/components/resources/ResourceRow.tsx [hunk 1]
```diff
@@ -13,29 +40,75 @@ const useStyles = makeStyles((theme: Theme) =>
...
+                            {name}
+                        </Typography>
+                        {version && (
+                            <Typography
+                                variant={'subtitle2'}
+                                className={classes.version}
+                                onClick={(): void => {
+                                    window.open(`https://www.npmjs.com/package/${packageName}`, '_blank');
+                                }}
+                            >{`@${version}`}</Typography>
```
Comment (huayunh) [inline src/app/components/resources/ResourceRow.tsx:104]: This click lacks some discoverability... fine with it for now, but I'd prob refine the design (like add an npm icon on the right)

**Review by huayunh — CHANGES_REQUESTED** [review]
Good job. Thanks, Evan!
* Why does ionic-design-pattern get displayed as a no-build?
* Code otherwise looks good

File: src/app/components/resources/ResourceRow.tsx [hunk 2]
```diff
@@ -13,29 +40,75 @@ const useStyles = makeStyles((theme: Theme) =>
...
                 style={{ paddingRight: theme.spacing(1) }}
-                divider={!small && props.divider ? 'full' : undefined}
-                title={props.title}
-                subtitle={props.description}
+                divider={!small && divider ? 'full' : undefined}
+                title={
+                    <div className={classes.flex} style={{ width: small ? '100%' : 'auto' }}>
+                        <Typography className={classes.title} noWrap>
+                            {packageName && xs ? packageName.replace('@pxblue/', '') : packageName}
+                            {name}
```
[the remark this answers is not in the document]
Comment (emclaug2) [reply src/app/components/resources/ResourceRow.tsx]: Good catch, I've just pushed out a fix for this.

**Review by huayunh — APPROVED** [review]

**Pull request merged**